code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -63,6 +73,11 @@
 		return
 	..()
 
+/obj/machinery/portable_atmospherics/canister/examine(user)
+	. = ..()
+	if(mode)
+		. += "A canister of tier [mode]"
```

Comment:
Missing span and period, same as before with the canister frame with this being added to the examine instead of replacing it.
```suggestion
		. += "<span class='notice'>This canister is tier [mode].</span>"
```